src/Makefile

```diff
@@ -118,6 +118,11 @@ uninstall:
 	test ! -f $(BINDIR)/atdgen.exe || rm $(BINDIR)/atdgen.exe
 	ocamlfind remove atdgen
 
+findlib-install:
+	ocamlfind install atdgen META \
+		 $(MLI) $(CMI) $(CMO) $(CMX) $(CMXS) $(O) \
+			atdgen.cma atdgen.a atdgen.cmxa
+
```

Comment:
@dsheets for me `make install` should install everything with reasonable defaults during a manual installation. Of course it's great to have sub-targets for fine-grained installs done by machines, by I want the `make install` to just do what a human unfamiliar with the software expects it to do.

I suggest the following targets: `install` (everything), `findlib-install`, `exe-install`, `uninstall` (everything), `findlib-uninstall`, `exe-uninstall`